documentation/configuration/metrics-configuration.md

```diff
@@ -290,6 +290,72 @@ For System.Diagnostics.Metrics, `dotnet monitor` allows you to set the maximum n
   ```
 </details>
 
+## Limit How Many Histograms To Track (8.0+)
```

Comment:
This looks like a duplication. I'm not sure how that happened.

Reply:
Yeah, that's odd. I'll close the PR, do a doc sync, and create a new PR with the new merge.